scripts/install-worker.sh

```diff
@@ -467,7 +467,8 @@ if [[ "$CACHE_CONTAINER_IMAGES" == "true" ]] && ! [[ ${ISOLATED_REGIONS} =~ $BIN
   #### Tag the pulled down image for all other regions in the partition
   for region in $(aws ec2 describe-regions --all-regions | jq -r '.Regions[] .RegionName'); do
```

Comment:
can you cache this at e.g. line 444 while you're at it? 😄 

```
REGIONS=$(aws ec2 describe-regions --all-regions --output text --query 'Regions[].[RegionName]')
...
for REGION in ${REGIONS[@]}; do
```

Reply:
updated in https://github.com/awslabs/amazon-eks-ami/pull/1442/commits/f58d7482049dd318223833e49b7b3ab691b0b050